flask_iiif/restful.py

```diff
@@ -75,7 +76,10 @@ def get(self, version, uuid):
             width, height = map(int, cached.split(','))
         else:
             data = current_iiif.uuid_to_image_opener(uuid)
-            image = IIIFImageAPIWrapper.open_image(data)
+            try:
+                image = IIIFImageAPIWrapper.open_image(data)
+            except:
+                raise(MultimediaImageNotFound)
```

Comment:
```suggestion
                raise MultimediaImageNotFound()
```

never saw `raise` called like this :) but it works! But perhaps the above way is more common.

Reply:
I will bow to convention :)